ibm/data_source_ibm_satellite_host_script.go

```diff
@@ -119,6 +119,8 @@ func dataSourceIBMSatelliteAttachHostScriptRead(d *schema.ResourceData, meta int
 				lines[i] = "yum update -y\nyum-config-manager --enable '*'\nyum repolist all\nyum install container-selinux -y"
 			} else if strings.ToLower(hostProvider) == "ibm" {
 				lines[i] = "subscription-manager refresh\nsubscription-manager repos --enable=*\n"
+			} else if strings.ToLower(hostProvider) == "vmware" {
+				lines[i] = "subscription-manager refresh\nyum update-y\n"
```

Comment:
Can we move this to an else statement so it’s performed for a provider that doesn’t have first class support?